pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc

```diff
@@ -0,0 +1,52 @@
+ifdef::context[:parent-context: {context}]
+
+include::../attributes.adoc[]
+
+[id="help"]
+= Help
+
+:context: assembly-pantheon-help
+
+Welcome to Pantheon help.
+
+This document describes the main concepts, requirements and, actions in Pantheon.
+
+include::modules/con_about_pantheon.adoc[leveloffset=+1]
+
+// Section 1
+== Modular documentation
+
+This section describes modular documentation.
```

Comment:
I believe Ingrid guides us to avoid intro sentences like this that don't really add any value.

Reply:
Changed as requested.